README.md

```diff
@@ -17,3 +17,15 @@ Command creates a random planar graph with the specified number of vertices base
 python main.py -random <seed> <number_of_vertices>>
 ```
 
+## Test
+Command runs tests:
```

Comment:
Maybe add a description where the tests save the results and what the results contain